CRM/Utils/Check.php

```diff
@@ -209,7 +212,7 @@ public static function checkAll($max = FALSE) {
 
     foreach ($messages as $key => $message) {
       $hush = self::checkHushSnooze($message);
-      $message->setVisible(!$hush);
+      $messages[$key]->setVisible(!$hush);
```

Comment:
Ah, ok good, I misunderstood your comment above.
I think it's fine as-is now. Thanks again for the quick fix.